sky/packages/sky_engine/lib/_embedder.yaml

```diff
@@ -1,26 +1,26 @@
-# This file is suitable for use within the tree. A different _embedder.yaml
-# is generated by the BUILD.gn in this directory. Changes here must be
-# mirrored there.
+# This file is generated by //flutter/sky/packages/sky_engine:_embedder_yaml
```

Comment:
I'm a bit confused by this comment. Is the behavior any different than before this change? If so, are there some changes to the BUILD.gn file that didn't get pushed?

Also, weren't there some other users of this file aside from the scenario apps? If not, and this file is only ever used out of the build output directory, and not directly, then this file is more like a template, and so we should rename it to something like `_embedder.yaml.tmpl`.

Reply:
Something is weird with my git checkout here - this file shouldn't hav ebeen modified.